plugins/modules/net_tools/dnsimple.py

```diff
@@ -201,113 +200,134 @@ def main():
     priority = module.params.get('priority')
     state = module.params.get('state')
     is_solo = module.params.get('solo')
+    pagination_per_page = 30
 
-    if account_email and account_api_token:
-        client = DNSimple(email=account_email, api_token=account_api_token)
-    elif os.environ.get('DNSIMPLE_EMAIL') and os.environ.get('DNSIMPLE_API_TOKEN'):
-        client = DNSimple(email=os.environ.get('DNSIMPLE_EMAIL'), api_token=os.environ.get('DNSIMPLE_API_TOKEN'))
-    else:
-        client = DNSimple()
+    try:
+        if account_email and account_api_token:
+            client = Client(email=account_email, access_token=account_api_token)
+        elif os.environ.get('DNSIMPLE_EMAIL') and os.environ.get('DNSIMPLE_API_TOKEN'):
```

Comment:
I reckon this is previously existing code, but instead of testing for the parameter and then testing for an env variable, we should be using `env_fallback`, as in: https://github.com/ansible-collections/community.general/blob/main/plugins/module_utils/alicloud_ecs.py#L40-L41

Reply:
done: https://github.com/ansible-collections/community.general/pull/2946/commits/3e6096d00fec2056dc06d8c7027c298888d3c370#diff-5d983a5b601bdc6628f8aa0911b85fe1c3a2c33038a0d35982b5fefce8c3a518R312-R315